.tekton/scripts/check-task-pipeline-bundle-repos.sh

```diff
@@ -27,48 +27,50 @@ locate_in_all_namespaces() {
 
     for quay_namespace in "${CATALOG_NAMESPACES[@]}"; do
         found=$(locate_bundle_repo "$quay_namespace" "$type" "$object")
+        echo "Checking ${quay_namespace}/${type}-${object}, http code: ${found}"
         if [ "$found" != "200" ]; then
-            echo "Missing $type bundle repo: ${quay_namespace}/${type}-${object}"
+            echo "Missing $type bundle repo: ${quay_namespace}/${type}-${object}, creating..."
+            echo "DEBUG: Creating quay repository https://quay.io/${quay_namespace}/${type}-${object}..."
+#            curl --oauth2-bearer "${QUAY_TOKEN}" 'https://quay.io/api/v1/repository' --json '{
+#              "namespace": '"${quay_namespace}"',
+#              "repository": "tekton-catalog/'"${type}-${object}"'",
+#              "visibility": "public",
+#              "description": ""
+#            }'
             rc=1
```

Comment:
The `rc` here should probably depend on the status code of the `curl` that creates the repo. And the script should fail if it fails to create a repo, e.g. due to missing permissions